BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +165,20 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Write out a help file for this Boutiques task
+      helpFileName = name + "_help.html" 
```

Comment:
Creating files in the code bas at run time is not allowed... what if the code is deployed under a UNIX user ID different from the USER that runs the Rails app ?